spotirec.py

```diff
@@ -374,6 +376,34 @@ def add_to_blacklist(entries: list):
         file.write(json.dumps(data))
 
 
+def remove_from_blacklist(entries: list):
+    """
+    Remove track(s) and/or artist(s) from blacklist.
+    :param entries: list of uris
+    """
+    try:
+        with open(blacklist_path, 'r') as file:
+            blacklist = json.loads(file.read())
+    except json.decoder.JSONDecodeError:
+        print('Error: blacklist is empty')
+        exit(1)
+    for uri in entries:
+        if 'track' in uri:
+            print(f'Removing track {blacklist["tracks"][uri]["name"]} by '
+                  f'{", ".join(str(x) for x in blacklist["tracks"][uri]["artists"]).strip(", ")} from blacklist')
+            del blacklist['tracks'][uri]
+        elif 'artist' in uri:
+            print(f'Removing artist \"{blacklist["artists"][uri]["name"]}\" from blacklist')
+            del blacklist['artists'][uri]
+        else:
+            print(f'uri \"{uri}\" is either not a valid uri for a track or artist, is malformed, or is not in '
+                  f'your blacklist')
+            print('Blacklist structure was recently re-done, so you may need to remove and re-do your blacklist. '
```

Comment:
```suggestion
            # FIXME: Remove this notice at some point
            print('Blacklist structure was recently re-done, so you may need to remove and re-do your blacklist. '
```
Hmm, while I think this is a nice touch, it should probably expire from the code-base at some point.

Hmm, while I think this is a nice touch, it should probably expire from the code-base at some point.

Reply:
That was the plan. Good idea adding that comment.